### ISSUE TITLE
Target selection is missing in automation for lights. and switch. domains (at least)

### ISSUE THREAD

**Issue by akrupnov** [home-assistant/frontend#22836]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

Target selection is missing in automations for lights. and switch. domain actions. See screenshot below - works fine for generic homeassistant action and for climate 
<img width="822" alt="image" src="https://github.com/user-attachments/assets/3493e90a-e1ae-49de-a8f5-c9cdac273f78">

### Describe the behavior you expected

I'd like to be able to chose entity id, area, label etc - same as for climate or homeassistant generic actio s

### Steps to reproduce the issue

1. Go to create new automation dialog
2. Click add action
3. Chose lights.turn_on

Expected: see targets and be able to chose them
Actual: empty UI

### What version of Home Assistant Core has the issue?

2024.11.1

### What was the last working version of Home Assistant Core?

Unknown

**Comment by akrupnov** [comment]
Looks like recently it was also reported on the community portal
https://community.home-assistant.io/t/missing-the-target-select-options-in-automation-editor/780241

**Comment by karwosts** [comment]
I've seen this reported from time to time, but it always ends up just resolving itself for users, or being some kind of issue with caching or some custom cards. 

You definitely tried the safe mode, & cleared caches?

In my install switch.turn_off definitely still has the target selectors.

Check the browser console for errors as well.

**Comment by akrupnov** [comment]
Yeah. Tried all that including totally never used browsers. Console is absolutely clean.
When I edit action yaml and switch it to homeassistant.turn_on I can edit targets and then change action again to lights via yaml but targets immediately disappear. 
I actually discovered this issue via another problem/quirk i was investigating for Node Red where they allegedly removed targets for actions on purpose "to align" with Homeassiatant https://github.com/zachowj/node-red-contrib-home-assistant-websocket/issues/1648#issuecomment-2412379934
